core-services/prow/02_config/openshift/ceph-csi/_pluginconfig.yaml

```diff
@@ -29,12 +29,12 @@ bugzilla:
               - status: POST
             master:
               exclude_defaults: true
-              target_release: OCS 4.8.0
+              target_release: OCS 4.9.0
             release-4.6:
-              target_release: OCS 4.6.0
+              target_release: OCS 4.6.7
               validate_by_default: true
             release-4.7:
-              target_release: OCS 4.7.2
+              target_release: OCS 4.7.3
               validate_by_default: true
             release-4.8:
```

Comment:
So, we can add a release-4.9 tag also, right?

Reply:
No, there is no release-4.9 branch in the current (`openshift/ceph-csi`) repo. At one point we'll need to copy/move this configuration over to the repo (`red-hat-data-services/ceph-csi`) where 4.9 will be maintained.

Reply:
Oh, yeah.. I forgot that :)